string name
string order_id
string city